docs/source/usecases/SoA-and-AoSoA-with-Cabana.md

```diff
@@ -52,8 +51,8 @@ class AoSoA;
 `DataTypes`
 : The types of the elements stored in the underlying `Cabana::SoA`s.
 
-`DeviceType`
-: The Kokkos device type that carries the information about where to execute code and where to allocate storage.
+`MemorySpace`
+: The Kokkos memory space that carries information about where to allocate storage.
 
 `VectorLength`
 : The vector length for the structure of arrays (optional).
```

Comment:
Maybe worth adding that while we have a default for each backend, the user will likely need to modify this for optimal kernel performance

Reply:
If you want to do "add a suggestion" I can merge into this PR.

Reply:
```suggestion
: The vector length for the structure of arrays (optional). If not specified, a default defined for each each memory space is used; this value will likely need to be modified for optimal performance.
```